include/glow/Optimizer/Optimizer.h

```diff
@@ -51,7 +52,13 @@ void lower(Function *F, CompilationContext &cctx, const Backend *B = nullptr,
            const KindSet &doNotLowerKinds = {});
 
 /// Dead code elimination.
-void DCE(Function *F);
+class DCE : public FunctionPass {
+private:
+  bool passImpl(Function *F);
+
+public:
+  const std::string getName() const { return "DCE"; }
```

Comment:
nit: why not returning string ref or const std::string&. We probably do not want copying string around too much.